src/newcharacter.cpp

```diff
@@ -2690,6 +2710,13 @@ tab_direction set_description( avatar &you, const bool allow_reroll,
                         you.mod_base_age( 1 );
                     }
                     break;
+                case char_creation::BLOOD:
+                    if( you.blood_type != AB_MINUS ) {
+                        you.blood_type = static_cast<blood_types>( static_cast<int>( you.blood_type ) + 1 );
+                    } else {
+                        you.blood_type = O_PLUS;
```

Comment:
```suggestion
                        you.blood_type = static_cast<blood_types>( 0 );
```
Suggest this rather than assuming `O_PLUS` is first.